psycopg/psycopg/_queries.py

```diff
@@ -33,11 +33,12 @@ class PostgresQuery:
 
     __slots__ = """
         query params types formats
-        _tx _want_formats _parts _encoding _order
+        _tx _want_formats _parts _encoding _order _raw_queries
         """.split()
 
-    def __init__(self, transformer: "Transformer"):
+    def __init__(self, transformer: "Transformer", raw_queries: Optional[bool] = False):
```

Comment:
Shouldn't this be:
```suggestion
    def __init__(self, transformer: "Transformer", raw_queries: bool = False):
```
?

I mean, how is the `None` value different from `False`?

Reply:
Yes, I agree, better to fix all users of PostgresQuery so they all pass raw_queries explicitly, perhaps then also the default can be skipped there, to only have one place where the default (= False) is specified.

Reply:
Fixed in d767b96